packages/cloud-cognitive/src/components/AddSelect/_add-select.scss

```diff
@@ -100,6 +100,17 @@
     }
   }
 
+  .#{$block-class}__selections-row:hover {
+    .#{$block-class}__selections-radio {
+      color: $interactive-04;
+    }
+
+    .#{$block-class}__selections-view-children svg path {
+      // stylelint-disable-next-line
+      fill: $interactive-04 !important;
```

Comment:
Any way to possibly work around the `!important`?

Reply:
yeah but it would just be a very long selector. but i can switch it.